binding/Binding/GRVkAlloc.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Runtime.InteropServices;
+
+namespace SkiaSharp
+{
+	using VkDeviceMemory = VkNonDispatchableHandle;
+	using VkDeviceSize = System.UInt64;
+
+	[StructLayout (LayoutKind.Sequential)]
+	public struct GRVkAlloc
+	{
+		public VkDeviceMemory Memory;
+		public VkDeviceSize Offset;
+		public VkDeviceSize Size;
+		public uint Flags;
```

Comment:
I would suggest making these fields private and then having properties to wrap them. This is both for consistency with the other types, and so that we can have more control over the order of fields and types of properties. Although this is not as fluid as skia, the API might change and we don't need to break backwards compatibility if we can avoid it. Unless if needs to be fields?